ci-operator/step-registry/cucushift/installer/check/ibmcloud/kms-key/cucushift-installer-check-ibmcloud-kms-key-ref.yaml

```diff
@@ -5,6 +5,16 @@ ref:
    name: "4.16"
    tag: upi-installer
   commands: cucushift-installer-check-ibmcloud-kms-key-commands.sh
+  env:
+  - name: IBMCLOUD_CONTROL_PLANE_ENCRYPTION_KEY
+    default: "true"
```

Comment:
The default value should be the same as https://github.com/openshift/release/pull/47656/files#diff-fa66772b4e70848940b69b72a1658edd9d5aaad86fffc4b05d8e00b3abe80c38R8